### PR TITLE
Updated new onboarding criteria

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation
Updated new onboarding criteria because sometimes 2 PRs can be small changes. And sometimes the contributor does not communicate well and will disappear soon. With that we see more new contributors so that we can keep growing the team, under the stricter acceptance criteria.

## Essential Checklist

- [ ] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

### CODE REVIEW

**Review by MohitGupta121 — COMMENTED** [review]
@adhiamboperes PTAL, Thanks!

File: wiki/Contributing-to-Oppia-Android.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ When you've found an issue you'd like to tackle:
 
 You are also welcome to tackle non-starter issues, but for those, you might need to be more independent, because we might not know how to solve them either!
 
-Once you've merged PRs that correspond to **two** different pre-existing GitHub issues, please fill in [this form](https://forms.gle/NxPjimCMqsSTNUgu5) to become an Oppia collaborator! This will grant you access to the repository, and allow you to join a team. (But please don't create your own issues and then make PRs for them -- that won't count.)
+Once you've merged PRs that correspond to **two** different pre-existing GitHub issues, please fill in [this form](https://forms.gle/NxPjimCMqsSTNUgu5) to be considered for a place on a team at Oppia android! A team lead will evaluate your contributions and give you access to the repository so that you can assign yourself to issues. (But please don't create your own issues and then make PRs for them -- that won't count.)
```
Comment (seanlip) [inline wiki/Contributing-to-Oppia-Android.md]: Oppia android --> Oppia Android

Comment (MohitGupta121) [reply wiki/Contributing-to-Oppia-Android.md]: Done.

**Review by adhiamboperes — APPROVED** [review]
Thanks @MohitGupta121  LGTM.

**Pull request merged**